README.md

```diff
@@ -369,11 +369,11 @@ Additionally, Moderators reserve the right to:
 Administrators outline and uphold our standards.
 
 * **Dispute regarding conduct**
-  * Administrators are permitted to resolve dispute regarding the code of conduct by using Context, Perspective and Interpretation to ensure order.
-* **Internel and Externel**
-  * Administrators are permitted to Investigate internel and externel interest and speak for the organization.
+  * Administrators are permitted to resolve dispute regarding the Code of Conduct by using Context, Perspective and Interpretation to ensure order.
+* **Internal and External**
+  * Administrators are permitted to investigate internal and external interests and speak for the organization.
```

Comment:
I'm going to be honest, "internal and external interests" is so nebulous to the point of being meaningless. This should be rewritten in more concrete terms.